### DISCUSSION TITLE
Highlight 99s and your lowest skill

### DISCUSSION THREAD

**Discussion by TalismanRS** [runelite/runelite#15189] [Ideas]
Would it be possible to add a plugin that highlights the grey background as green if the skill is 99, and red if its your lowest skill?

I'll give a super quickly made example to show what I'm talking about :)

![Capture](https://user-images.githubusercontent.com/73463113/154891366-e3dc92bf-8856-4d8c-a796-266d5106fe62.PNG)

**Comment by adwitkow** [comment]
The lowest skill part is already covered by the [ToG indicator](https://github.com/LlemonDuck/tog-indicator) external plugin.
